felix/daemon/daemon.go

```diff
@@ -408,6 +406,10 @@ configRetry:
 		simulateDataRace()
 	}
 
+	if configParams.DebugPort != 0 {
+		debugserver.StartDebugPprofServer(configParams.DebugHost, configParams.DebugPort)
```

Comment:
what if debughost is something bogus? Should it log an error here like "we did not start the debug server because ..." ?

Reply:
I don't think we know until we call ListenAndServe, which is on the background goroutine